projects/storefrontlib/src/cms-components/user/login/login.component.spec.ts

```diff
@@ -111,21 +111,26 @@ describe('LoginComponent', () => {
 
   it('should have user details when token exists', () => {
     let user;
+    component.ngOnInit();
```

Comment:
maybe this `ngOnInit` can go to `beforeEach`.

Reply:
Fixed.  However, `component.ngOnInit();` still needs to be called again after the spy statements.